### PR TITLE
Renamed histEqual to histEqualize.

### PR DESCRIPTION
**Pull request by ljubobratovicrelja**
Working on #76

I left histEqual with deprecation message. Will leave the issue until 0.2.0 version comes, and until we remove the histEqual. 

@9il seems ok?

### CODE REVIEW

File: source/dcv/imgproc/filter.d [hunk 1]
```diff
@@ -986,6 +986,30 @@ body
...
+    Histogram = (template parameter) Histogram type, see $(LINK2 #calcHistogram,calcHistogram) function for details.
+    slice = Input image slice.
+    histogram = Histogram values for input image slice.
+    prealloc = Optional pre-allocated buffer where equalized image is saved.
+
+Returns:
+    Copy of input image slice with its histogram values equalized.
+*/
+deprecated("Use dcv.imgproc.filter.histEqualize")
+Slice!(N, T*) histEqual(T, HistogramType, size_t N)(Slice!(N, T*) slice, HistogramType histogram,
```
Comment (9il) [inline source/dcv/imgproc/filter.d]: `alias histEqual = histEqualize`

Comment (ljubobratovicrelja) [reply source/dcv/imgproc/filter.d]: Wanted to do this, but how to inform the deprecation? Is it possible to deprecate alias?

Comment (9il) [reply source/dcv/imgproc/filter.d]: yes

Comment (ljubobratovicrelja) [reply source/dcv/imgproc/filter.d]: Awesome! Will do..

**Comment by 9il** [comment]
LGTM

**Comment by 9il** [comment]
See also dcv/tests/performance-tests/source/performance/measure.d

**Comment by ljubobratovicrelja** [comment]
> See also dcv/tests/performance-tests/source/performance/measure.d

ah, tnx..

**Comment by ljubobratovicrelja** [comment]
> See also dcv/tests/performance-tests/source/performance/measure.d

I forgot about this one in bilateralFilter sigma separation PR, so I smuggled it here. Hope you don't mind. (it was failing to compile the benchmark app)

**Pull request merged**